minor amendments to the parameters guide (#182)
OAI/Documentation: specification/parameters.md

### Parameter Location

> Parameters can reside in different locations, indicated by the `in` field. The different locations are:

Reviewer: In the previous version, were we deliberately implying that there are other locations than these? Because that seems to be lost in this update. I can't think of another location...

  ```

  > **NOTE**:

>   > If any parameters are `in: query`, then `in: querystring` cannot be used in the same endpoint.

Reviewer: ```suggestion
  > For each operation, if there are any parameters declared with `in: query`, then `in: querystring` cannot be used.
```

Maybe this?
